kafka-connect-s3/src/test/java/io/confluent/connect/s3/util/UtilsTest.java

```diff
@@ -76,7 +83,8 @@ public void getAdjustedFilenameForHeaders() {
         "header.headers.avro",
         "sample-filename.headers.avro",
         "sample.filename.headers.avro",
-        "sample.file.name.headers.avro"
+        "sample.file.name.headers.avro",
+        "sample.file.avro.RawEvent.headers.avro"
```

Comment:
Nit: I'd add a couple more test cases: 1. the extension is present in the middle of the name (but not at the end) 2. the filename is empty, it's just the extension.

Reply:
@erdody Good idea, please check now.

Reply:
@kpatelatwork looks good, thx